script/update_beats.sh

```diff
@@ -28,6 +28,7 @@ rsync -crpv --delete \
     --exclude="dev-tools/jenkins_intake.sh" \
     --exclude="dev-tools/packaging/preference-pane/***" \
     --exclude="dev-tools/mage/***" \
+    --exclude="libbeat/processors/script/***" \
```

Comment:
nit: would you mind moving this down to the `libbeat/processors` section for better readability?

Reply:
I think excludes need to go before includes to resolve conflicts properly, so it is better to group all the excludes at the top to avoid risking wrong order